edx_exams/apps/api/serializers.py

```diff
@@ -16,7 +16,7 @@ class ExamSerializer(serializers.ModelSerializer):
     course_id = serializers.CharField(required=False)
     content_id = serializers.CharField(required=True)
     time_limit_mins = serializers.IntegerField(required=True)
-    due_date = serializers.DateTimeField(required=False, format=None)
+    due_date = serializers.DateTimeField(required=True, format=None, allow_null=True)
```

Comment:
the view will blow up if this property doesn't exist so I think changing this to be required but allow null is prudent.